transaction-overview.md

```diff
@@ -62,42 +72,104 @@ Syntax:
 ROLLBACK;
 ```
 
-You can use this statement to roll back the current transaction and cancels all updates between `[BEGIN | START TRANSACTION]` and `ROLLBACK`.
+Transactions are also automatically rolled back if the client connection is aborted or closed.
 
 ## Autocommit
 
-Syntax:
+As part of MySQL compatibility, TiDB supports the ability to _autocommit_ a statement immediately following its execution.
 
-{{< copyable "sql" >}}
+i.e.
 
 ```sql
-SET autocommit = {0 | 1}
+mysql> CREATE TABLE t1 (
+    ->  id INT NOT NULL PRIMARY KEY auto_increment,
+    ->  pad1 VARCHAR(100)
+    -> );
+Query OK, 0 rows affected (0.09 sec)
+
+mysql> SELECT @@autocommit;
++--------------+
+| @@autocommit |
++--------------+
+| 1            |
++--------------+
+1 row in set (0.00 sec)
+
+mysql> INSERT INTO t1 VALUES (1, 'test');
+Query OK, 1 row affected (0.02 sec)
+
+mysql> ROLLBACK;
+Query OK, 0 rows affected (0.01 sec)
+
+mysql> SELECT * FROM t1;
++----+------+
+| id | pad1 |
++----+------+
+|  1 | test |
++----+------+
+1 row in set (0.00 sec)
 ```
 
-When `autocommit = 1` (default), the status of the current session is autocommit. That is, statements are automatically committed immediately following their execution.
+In the above example, the `ROLLBACK` statement has no effect. This is because the `INSERT` statement executed in autocommit. That is, it was the equivalent of the following single-statement transaction:
```

Comment:
```suggestion
In the above example, the `ROLLBACK` statement has no effect. This is because the `INSERT` statement is executed in autocommit. That is, it was the equivalent of the following single-statement transaction:
```